src/metabase/api/card.clj

```diff
@@ -1148,46 +1158,63 @@ saved later when it is ready."
   (when (not (public-settings/uploads-enabled))
     (throw (Exception. "Uploads are not enabled.")))
   (collection/check-write-perms-for-collection collection-id)
-  (let [db-id             (public-settings/uploads-database-id)
-        database          (or (t2/select-one Database :id db-id)
-                              (throw (Exception. (tru "The uploads database does not exist."))))
-        driver            (driver.u/database->driver database)
-        schema-name       (public-settings/uploads-schema-name)
-        _check-schema     (when-not (or (nil? schema-name)
-                                        (driver.s/include-schema? database schema-name))
-                            (throw (ex-info (tru "The schema {0} is not syncable." schema-name)
-                                            {:status-code 422})))
-        filename-prefix   (or (second (re-matches #"(.*)\.csv$" filename))
-                              filename)
-        _check-setting    (when-not (driver/database-supports? driver :uploads nil)
-                            (throw (Exception. (tru "Uploads are not supported on {0} databases." (str/capitalize (name driver))))))
-        table-name        (->> (str (public-settings/uploads-table-prefix) filename-prefix)
-                               (upload/unique-table-name driver)
-                               (u/lower-case-en))
-        schema+table-name (if (str/blank? schema-name)
-                            table-name
-                            (str schema-name "." table-name))
-        _load!            (upload/load-from-csv driver db-id schema+table-name csv-file)
+  (try (let [start-time        (System/currentTimeMillis)
```

Comment:
```suggestion
  (try 
    (let [start-time        (System/currentTimeMillis)
```

most of the time I see the body of try is on a new line, let's follow that convention